### PR TITLE
Add content model to callout with media

### PR DESCRIPTION
**Pull request by mfearer94**
Added an anchor link and content model to the callout with media component page

### Description

Added the content model to this page so users can see character lengths and other details on the component

### Changelog

**New**

- Anchor link to content guidance
- Content model

### CODE REVIEW

**Review by JennySanchez — APPROVED** [review]
Looks good to me. :-)

File: src/pages/components/callout-with-media.mdx [hunk 1]
```diff
@@ -23,9 +24,19 @@ Callout &mdash; with media uses high contrast colors to create a delightful paus
 
 ![Default](../../images/pattern/callout-with-media/callout-with-media.jpg)
 
+## Content guidance
+
+| Field | Field type | Required   | Cardinality | Max character size (English / translated) |
+| -------- | --------- | ---------- | ---------- | ----------|
+| Headline | Short copy | Yes   | 1 | XXS (40 / 55)|
+| Paragraph | Long copy | No  | 1 | XXL (350 / 450) |
+| [Video player](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/video-player) or [Image with caption](https://www.ibm.com/standards/web/carbon-for-ibm-dotcom/components/image-with-caption) | Component | Yes | 1 | - |
```
Comment (photodow) [inline src/pages/components/callout-with-media.mdx:33]: Should be the same as `content block — simple` if we'd like to link out. If you decide to maintain images should be allowed here as well.

File: src/pages/components/callout-with-media.mdx [hunk 2]
```diff
@@ -23,9 +24,19 @@ Callout &mdash; with media uses high contrast colors to create a delightful paus
...
+
+To see the list of all character count standards, go to this <a href="https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Character-count-standards" target="_blank" rel="noreferrer noopener nofollow">link</a>.
```
Comment (photodow) [inline src/pages/components/callout-with-media.mdx:35]: I added a comment to a previous PR around this. I'll stop commenting on it for this round, and assume you'll take a look across the aboard at all your PRs.

**Review by photodow — COMMENTED** [review]
One comment. Let me know your thoughts before I approve.

File: src/pages/components/callout-with-media.mdx (hunk 1 above)
Comment (mfearer94) [reply src/pages/components/callout-with-media.mdx:33]: @photodow I'm wondering if we should just mention below the table that this component refers to "content block - simple". That way we are referring to it, but the information is also here. Thoughts?

File: src/pages/components/callout-with-media.mdx (hunk 2 above)
Comment (mfearer94) [reply src/pages/components/callout-with-media.mdx:35]: 👍

File: src/pages/components/callout-with-media.mdx (hunk 1 above)
Comment (photodow) [reply src/pages/components/callout-with-media.mdx:33]: That might work for now, but I'm not sure it gets at the crux of maintaining the same data in multiple places.

**Pull request merged**